src/index.js

```diff
@@ -1 +1,12 @@
 import './style.css';
+import renderData from './utils/get.js';
+import recieveData from './utils/post.js';
+
+const refreshBtn = document.querySelector('.refresh-btn');
+
+refreshBtn.addEventListener('click', () => {
+  window.location.reload();
+});
+
```

Comment:
- [ ] The refresh button should receive data from the API and parse the JSON causing the scores list to change, kindly modify the event listener to call the renderData function on click.